README.md

```
@@ -22,141 +37,189 @@ Regula supports the following file types:
 
-1. [Install Go (v1.18+)](https://go.dev/doc/install)
+---
 
-2. Build binary and move to `/usr/local/bin/regula`:
+## Available Commands
 
-    ```bash
-    make # this builds ./bin/regula
-    make install # this builds ./bin/regula and installs it to /usr/local/bin/regula
-    ```
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)